### PR TITLE
[IBM MQ] Update changelog entry for automatic promotion of package to snapshot

### PR DESCRIPTION
**Pull request by harnish-crest**
- Bug

## What does this PR do?

Update changelog entry for automatic promotion of package to snapshot.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by yug-elastic** [comment]
CC: @cachedout 
This PR is the outcome of our discussion. Could you please take a look? It just contains a version bump up, and does not contain any other functional/non-functional changes.

**Review by yug-elastic — APPROVED** [review]
LGTM

**Review by cachedout — APPROVED** [review]
This looks like what we discussed. Let's see what the CI has to say about it.

File: packages/ibmmq/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 0.1.1
+  changes:
+    - description: No change in the package, bumping up the version to have the latest package in snapshot
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/3970
 - version: 0.1.0
   changes:
     - description: IBM MQ integration package with "qmgr" data stream
```
Comment (jsoriano) [inline packages/ibmmq/changelog.yml:9]: As this change was not released in 0.1.0, please move this entry to 0.1.1 and remove the current placeholder entry there.

**Review by jsoriano — CHANGES_REQUESTED** [review]

**Review by jsoriano — APPROVED** [review]

File: packages/ibmmq/changelog.yml (hunk 1 above)
Comment (harnish-crest) [reply packages/ibmmq/changelog.yml:4]: Thanks a lot for the help!

**Pull request merged**